test/conftest.py

```diff
@@ -56,26 +55,51 @@ def pytest_generate_tests(metafunc):
     Based on examples from https://docs.pytest.org/en/latest/example/parametrize.html.
     """
 
-    for mark in getattr(metafunc.function, 'pytestmark', []):
-        if getattr(mark, 'name', '') == 'test_all_runtimes':
+    for mark in getattr(metafunc.function, "pytestmark", []):
+        if getattr(mark, "name", "") == "test_all_runtimes":
             args = tuple(
                 pytest.param(
                     runtime,
                     marks=pytest.mark.skipif(
                         shutil.which(runtime) is None,
-                        reason=f'{runtime} is not installed',
+                        reason=f"{runtime} is not installed",
                     ),
                 )
                 for runtime in CONTAINER_RUNTIMES
             )
-            metafunc.parametrize('runtime', args)
+            metafunc.parametrize("runtime", args)
             break
 
 
+@pytest.fixture(scope="session")
```

Comment:
This is the real addition, not sure how I feel about subprocess here but we need to get ansible's internal core version